CartPole/cartpole_model.py

```diff
@@ -1,16 +1,22 @@
 from types import SimpleNamespace
 from typing import Union
-from CartPole.state_utilities import (
-    create_cartpole_state,
-    ANGLE_IDX, ANGLED_IDX, POSITION_IDX, POSITIOND_IDX, ANGLE_COS_IDX, ANGLE_SIN_IDX
-)
-
-from others.p_globals import (
-    k, M, m, g, J_fric, M_fric, L, v_max, u_max, controlDisturbance, controlBias, TrackHalfLength
-)
 
 import numpy as np
-from numpy.random import SFC64, Generator
+import os
+import yaml
+from others.globals_and_utils import create_rng
+from others.p_globals import (J_fric, L, M, M_fric, TrackHalfLength,
+                              controlBias, controlDisturbance, g, k, m, u_max,
+                              v_max)
+
+from CartPole.state_utilities import (ANGLE_COS_IDX, ANGLE_IDX, ANGLE_SIN_IDX,
+                                      ANGLED_IDX, POSITION_IDX, POSITIOND_IDX,
+                                      create_cartpole_state)
+
+config = yaml.load(open(os.path.join(os.path.dirname(__file__), "..", "config.yml"), "r"), Loader=yaml.FullLoader)
```

Comment:
Check

Reply:
Maybe this config loading is also worth of a helper function as it comes so often... Not sure.